ci/localnet-sanity.sh

```diff
@@ -156,19 +156,7 @@ startNodes() {
   initCompleteFiles=()
   maybeExpectedGenesisHash=
   for i in $(seq 0 $((${#nodes[@]} - 1))); do
-    declare cmd=${nodes[$i]}
-
-    declare initCompleteFile="init-complete-node$i.log"
-    rm -f "$initCompleteFile"
-    initCompleteFiles+=("$initCompleteFile")
-
-    startNode "$i" "$cmd $maybeExpectedGenesisHash"
-    if $addLogs; then
-      logs+=("$(getNodeLogFile "$i" "$cmd")")
-    fi
-
-    # 1 == bootstrap validator, wait until it boots before starting
-    # other validators
+    # wait for bootstrap validator to boot before starting other validators
```

Comment:
nit: Maybe explicitly mention that bootstrap == index 0.

Also, this might have "worked" at one point. Passing `initCompleteFile` to `waitForNodToInit` is tricky because the value (if what integer is attached to the log file) is getting updated in the loop

Maybe we add a new variable to make the intent a little more clear. Something like:
```bash
if [[ "$i" -eq 1 ]]; then
    declare bootstrapInitCompleteFile="init-complete-node0.log"
    waitForNodeToInit "$bootstrapInitCompleteFile"
...
```